ament_flake8/ament_flake8/main.py

```diff
@@ -285,5 +288,21 @@ def print_statistics(self):
         self.report._application.report_statistics()
 
 
+class ReturnCode(int):
```

Comment:
Adding a `__repr__` method can maybe be useful.

Reply:
What do you propose that method to return?

Reply:
I imagine this being used like:
```python3
if ret_code != 0:
    print(ret_code)
```

So, I would return something like:
```python3
def __repr__(self):
        return f"ReturnCode(): code='{str(super())}' error='{self.error_strings}'"
```